flow/react-native-host-hooks.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @flow
+ */
+
+/* eslint-disable */
+
+declare module 'deepDiffer' {
+  declare function exports(one: any, two: any): bool;
+}
+declare module 'deepFreezeAndThrowOnMutationInDev' {
+  declare function exports<T>(obj : T) : T;
+}
+declare module 'flattenStyle' { }
```

Comment:
Because that particular line is FlowFixMe anyway for other reasons. I'm not too worried since I have a follow up for decoupling or moving these anyway. https://gist.github.com/sebmarkbage/b6f7a9c4bdec1c41cb64e7ee77254551